### ISSUE TITLE
Model training with the released synthetic CIFAR-10 (ipc=50)

### ISSUE THREAD

**Issue by LeavesLei** [GeorgeCazenavette/mtt-distillation#29]
Hi George,

I attempt to directly use the release synthetic dataset of CIFAR-10 (ipc=50) to train the default ConvNet, but the final test accuracy is only 55.67% on the orinigal test set of CIFAR-10. The training details are as follows:

- I have not do any transformation w.r.t. the synthetic images (btw, the CIFAR-10 (IPC-50) does not seem to require ZCA preprocessing). Address of downloading the dataset is https://georgecazenavette.github.io/mtt-distillation/tensors/index.html#tensors/cifar10_50
- - Augmentation is used for synthetic images to train the nework as the same as it shown in the function of `augment()`
- architecture: default ConvNet
- test set preprocessing: transforms.ToTensor() and transforms.Normalize(mean, std)
- learning rate: initial lr=0.01 and decays to 0.001 at the 500th epoch
- the number of training epochs: 1000
- optimizer: SGD with momentum=0.9 and weight_decay=5e-4
- loss function: cross entropy
- GPU: NVIDIA TITAN V

Do I miss other implementation details?

Best,
Shiye

**Comment by Guang000** [comment]
Maybe you should use ConvNetD3 :)

**Comment by rave78** [comment]
I think the evaluation of synthetic data is always with DiffAugment (Line 322](https://github.com/GeorgeCazenavette/mtt-distillation/blob/c365f4257117ccc0abf163e09f692be94634ed18/utils.py#L322) because `args.dsa` is True by default. Also, did you consider to normalize with ZCA

**Comment by GeorgeCazenavette** [comment]
Hi all, apologies for the late reply.

Where you able to resolve this on your own? 

@rave78 is correct, we did use DiffAugment during evaluation.

There is definitely a chance that I uploaded the wrong dataset file. I can double check if you're still having issues.

**Comment by Namelessness5** [comment]
Hi,
I wonder if you guys uploaded the wrong dataset file. I am also trying to reproduce this experiment and I got around 67% accuracy in test set which is still lower than reported data. I applied DiffAugment and trained for 2000 epochs.
